### PR TITLE
fix and generate docs for FusedRMSNorm

### PR DESCRIPTION
**Pull request by eqy**
Followup to #1274, which left out (correct) documentation

### CODE REVIEW

**Review by crcrpar — APPROVED** [review]
The generated docs look nice

image

File: apex/normalization/fused_layer_norm.py [hunk 1]
```diff
@@ -303,19 +303,19 @@ class FusedRMSNorm(torch.nn.Module):
     Currently only runs on cuda() tensors.
 
     .. math::
-        y = \frac{x - \mathrm{E}[x]}{ \sqrt{\mathrm{Var}[x] + \epsilon}} * \gamma + \beta
+        y = \frac{x}{\mathrm{RMS}[x]} * \gamma
```
Comment (stas00) [inline apex/normalization/fused_layer_norm.py:306]: It's missing epsilon and I checked it's in the code.

Comment (stas00) [reply apex/normalization/fused_layer_norm.py:306]: https://github.com/NVIDIA/apex/pull/1295
